src/apps/dfx/ethereum/dfx.balance-fetcher.ts

```diff
@@ -0,0 +1,38 @@
+// Test address: http://localhost:5001/apps/dfx/balances?addresses[]=0x1867608e55a862e96e468b51dc6983bca8688f3d&network=ethereum
+//
```

Comment:
Comments not needed, you can add this to the PR description if you'd like for future reference.

Reply:
Fix https://github.com/Zapper-fi/studio/pull/347/commits/e2764f586d434f45e826f2657b5b1acdc812ae10

Added Polygon test addresses to original template post